notebooks/HASP/FluxScaleTutorial/FluxScaleTutorial.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<p>The beginning of the sentence contains a duplication of the word "code" in the Hubble Advanced Spectral Products (HASP) code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/172/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>